gulpfile.js

```diff
@@ -137,4 +139,8 @@ gulp.task('patch', function() {
     runSequence('bump-patch', 'build', 'update-version');
 });
 
-gulp.task('build', ['css', 'js', 'css-min', 'js-min']);
+gulp.task('build', function() {
+    runSequence('lint', 'css', 'js', 'css-min', 'js-min', 'update-version')
```

Comment:
Well, like I have said, I'm not familiar with Gulp. Adn css-min depends on css the same as js-min depends on js.